lib/main.coffee

```diff
@@ -1,4 +1,5 @@
 {CompositeDisposable} = require 'atom'
+escapeHtml = require 'escape-html'
```

Comment:
Move this into the `provideLinter`, otherwise it will be required on package activation.